sdk/src/azure/storage/az_storage_blobs_blob_client.c

```diff
@@ -54,63 +54,60 @@ AZ_INLINE az_span _az_get_host_from_url(az_span url)
   int32_t const url_length = az_span_size(url);
   uint8_t const* url_ptr = az_span_ptr(url);
 
-  if (url_length >= (int32_t)(sizeof("s://h") - 1))
```

Comment:
GitHub is not displaying this change visually clear. What this all is about, is to remove the check that is verified by a precondition below.

<img width="551" alt="image" src="https://user-images.githubusercontent.com/41349689/148825207-dd926a5f-a194-44d4-ab4b-4f44510cef47.png">

Reply:
Can we separate out this change into another PR.  That way this PR is purely about improving code coverage via additional tests.

Reply:
@RickWinter, done, here it is - https://github.com/Azure/azure-sdk-for-c/pull/2062